src/app/tests/ConnectWire.test.ts

```diff
@@ -0,0 +1,34 @@
+import {V} from "Vector";
+
+import {CreateCircuit} from "digital/public";
+
+import "./Extensions";
+
+
+describe("ConnectWire", () => {
+    test("Basic Connect Wire Different Ports", () => {
+        const circuit = CreateCircuit();
+        const c = circuit.placeComponentAt(V(0, 0), "ANDGate");
+        const l1 = circuit.placeComponentAt(V(5, 0), "LED");
+        expect(c.pos).toEqual(V(0, 0));
+        expect(l1.pos).toEqual(V(5, 0));
```

Comment:
you don't need to test that the positions are correct, we're just testing `connectWire` here